assets/scss/common/top-bar.scss

```diff
@@ -388,6 +375,10 @@ $topbar-dropdown-arrows: false !default; //Set false to remove the \00bb >> text
           & > .dropdown {
             @include topbar-show-dropdown();
             width: 100%;
+            top: 0;
+            &:before {
+              display: none;
```

Comment:
@tahb There is a triangle indicator when you hover over a link with dropdown (go to https://default.govsite.com/ and hover over Publications) which I had to remove on mobile view.